Makefile

```diff
@@ -29,5 +29,5 @@ lint:
 
 # this will build the containers and then try to use them to build themselves again, making sure we didn't break docker support
 build-containers: maistra-builder
```

Comment:
don't you have to change the target above too?

Reply:
or... actually the dependency

Reply:
no. we still want to build all containers. what's changing is how many containers we build to test docker-in-docker